modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +97,124 @@ Status description:: error: connection exception - alias chains are not permitte
+
+=== 22N11 
+
+Status description:: error: data exception - invalid argument. Invalid argument: cannot process { $input }.
+
+=== 22N12 
+
+Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process { $input }.
+
+=== 22N13 
+
+Status description:: error: data exception - invalid time zone. Specified time zones must include a date component.
+
+=== 22N14 
+
+Status description:: error: data exception - invalid temporal value combination. Cannot select both { $temporal } and { $component }.
+
+=== 22N15 
+
+Status description:: error: data exception - invalid temporal component. Cannot read the specified { $component } component from { $temporal }.
+
+=== 22N16 
+
+Status description:: error: data exception - invalid import value. Importing entity values to a graph with a USE clause is not supported. Attempted to import { $expr } to { $graph }.
+
+=== 22N17 
+
+Status description:: error: data exception - invalid date, time, or datetime function field name. Cannot read the specified { $component } component from { $temporal }.
+
+=== 22N18 
+
+Status description:: error: data exception - incomplete spatial value. A { $crs } POINT must contain { $mapKeyAndedList }.
+
+=== 22N19 
+
+Status description:: error: data exception - invalid spatial value. A POINT must contain either 'x' and 'y', or 'latitude' and 'longitude'.
+
+=== 22N20 
+
+Status description:: error: data exception - invalid spatial value dimensions. Cannot create POINT with { $crs } coordinate reference system (CRS) and { $coordinates } coordinates. Use the equivalent { $crs2 } coordinate reference system instead.
+
+=== 22N21 
+
+Status description:: error: data exception - unsupported coordinate reference system. Unsupported coordinate reference system (CRS): { $crs }.
+
+=== 22N22 
+
+Status description:: error: data exception - invalid spatial value combination. Cannot specify both coordinate reference system (CRS) and spatial reference identifier (SRID).
+
+=== 22N23 
+
+Status description:: error: data exception - invalid latitude value. Cannot create WGS84 POINT with invalid coordinate: { $coordinates }. The valid range for the latitude coordinate is [-90, 90].
+
+=== 22N24 
+
+Status description:: error: data exception - invalid coordinate arguments. Cannot construct a { $valueType } from { $coordinates }.
+
+=== 22N25 
+
+Status description:: error: data exception - invalid temporal arguments. Cannot construct a { $valueType } from { $temporal }.
+
+=== 22N26 
+
+Status description:: error: data exception - unsupported rounding mode. Unknown rounding mode. Valid values are: CEILING, FLOOR, UP, DOWN, HALF_EVEN, HALF_UP, HALF_DOWN, UNNECESSARY.
+
+=== 22N27 
+
+Status description:: error: data exception - invalid entity type. Invalid input { $input } for { $var }. Expected to be one of { $valueTypeOredList }.
```

Comment:
```suggestion
Status description:: error: data exception - invalid entity type. Invalid input { $input } for { $variable }. Expected to be one of { $valueTypeOredList }.
```

Again, something which we seem to have changed in the code but not the spreadsheet. 'var' is a reserved word in Java/Scala so it was very impractical to use as a variable name, therefore we changed it to 'variable'